models/UserDao.js

```diff
@@ -0,0 +1,9 @@
+import prisma from '../prisma';
+
+const getUser = async (user_account) => {
+  return await prisma.$queryRaw(`
+    SELECT * FROM users WHERE user_account='${user_account}';
```

Comment:
```suggestion
import prisma from '../prisma';

const getUser = async (userAccount) => {
  return await prisma.$queryRaw(`
    SELECT * FROM users WHERE user_account='${userAccount}';
```
데이터베이스와 맞춰 주시기 위해서 `snake_case`를 사용하신 것 같아요!
하지만 함수 인자는 자바스크립트 컨벤션에 맞춰서 camelCase사용해주시면 됩니다 ❤️

Reply:
으아앗 소헌님 캄사합니다🙈💖
수정했습니다!!!!!!

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-backend/pull/2/commits/eda4fc9dd33da6b387b80c5a453b79d38db6a2a4